<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_examinationInfo</class>
 <widget class="QWidget" name="form_examinationInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>384</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>384</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Исследование</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_clientPic">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../rec.qrc">:/resources/user_img.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_clientSurname">
         <property name="font">
          <font>
           <family>Calibri Light</family>
           <pointsize>14</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;Surname&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_clientName">
         <property name="font">
          <font>
           <family>Calibri Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;Name&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_clientPatronymic">
         <property name="font">
          <font>
           <family>Calibri Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;Patronymic&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Возраст:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_clientAgeYear">
           <property name="font">
            <font>
             <family>Calibri Light</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>xx (xxxx)</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Информация об исследовании</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Тип:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_examinationType">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;examination_type&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Дата:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label__examinationDateTime">
       <property name="font">
        <font>
         <family>Calibri Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>dd.mm.yyyy hh:mm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Действия</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPushButton" name="pushButton_examinationEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Редактировать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_examinationDelete">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Экспорт</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_examinationHalfExport">
          <property name="text">
           <string>Сокращенная форма</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_examinationFullExport">
          <property name="text">
           <string>Полная форма</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rec.qrc"/>
 </resources>
 <connections/>
</ui>
